plugins/BEdita/Core/src/Utility/ResourcesBase.php

```diff
@@ -47,8 +47,8 @@ protected static function getTable(string $type, array $options = []): Table
             );
         }
         $type = Inflector::camelize($type);
-        TableRegistry::getTableLocator()->remove($type);
+        $tl = new TableLocator();
```

Comment:
Maybe it should be better to set our `TableLocator` in TableRegistry` as we do in the core bootstrap.

```php
if (!TableRegistry::getTableLocator() instanceof TableLocator) {
    TableRegistry::setTableLocator(new TableLocator());
}
```